ref/feature/transportSecurity/examples.adoc

```diff
@@ -75,27 +86,36 @@ You can set the `trustDefaultCert` attribute to `true` to specify that the JVM d
 
 You can configure Liberty to have a global outbound default TLS configuration that is different from the inbound default configuration. Outbound TLS filters are configured on the `outboundConnection` element that is nested in the `ssl` element. You can specify either a host or a host and port to indicate where the outbound SSL connection goes by setting attributes on the `outboundConnection` element.
 
-In the following example, outbound TLS connections that go to any port on the `otherhost` host use the `defaultSSLConfig` TLS configuration. Outbound connections that go to the `server99.ibm.com` host and either port 9443 or 8020 use the `alternateSSLSettings` TLS configuration. All other outbound TLS connections use the outbound default TLS configuration that is specified by the `outboundSSLRef` attribute called `outboundSSLSettings`.
+In the following example, outbound TLS connections that go to any port on the `otherhost` host use the `defaultSSLConfig` All other outbound TLS connections use the outbound default TLS configuration that is specified by the `outboundSSLRef` attribute called `outboundSSLSettings`.
```

Comment:
'In the following example, outbound TLS connections that go to any port on the `otherhost` host use the `defaultSSLConfig`' Add a period